src/loader.ts

```diff
@@ -54,11 +56,45 @@ export class Loader {
 
     container.appendChild(this.renderer.domElement);
     
-    this.measurement = new Measurements(this.scene);
+    this.measurement = new Measurements(this.scene, this.camera, this.renderer, this.group, this);
```

Comment:
```suggestion
    this.measurement = new Measurements(this.scene, this.camera, this.renderer, this.group);
```

Handle all events in drag-control-handler